### DISCUSSION TITLE
Is there a way to return json according to UnderscoreNamingStrategy of mysql?

### DISCUSSION THREAD

**Discussion by AlexandrosMelissas** [mikro-orm/mikro-orm#2907] [Q&A] [answered]
Hello. Let's say we have an entity property called "createdAt". By default, using mysql driver, this property will be saved as "created_at" in db (using the default UnderscoreNamingStrategy) and will be returned to the client as "createdAt". Is there a way to return it as "created_at" just as it is in the db without actually renaming the property in mikro-orm? If yes, is there a way to do such thing globally?

**Comment by B4nan** [comment, marked as answer]
Return from where? You mean after serialization? You could use [`serializedName`](https://mikro-orm.io/docs/serializing#property-serializers) option for that, I am not aware of any way to handle that globally.

**Reply by AlexandrosMelissas** [reply]
Ahh i see. That pretty much got me covered. Thank you.

**Reply by B4nan** [reply]
Btw with a bit of metadata magic, you could do this globally too, something like following could work:

```ts
const orm = await MikroORM.init(...);

for (const meta of Object.values(orm.getMetadata().getAll())) {
  meta.props.forEach(prop => {
    prop.serializedName = prop.fieldNames[0];
  });
}
```

(note this won't work for composite FKs, where `prop.fieldNames` will contain more than one element)
